image/cinc/install_ubuntu_pkg.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -xe
+
+DEBIAN_FRONTEND=noninteractive
+
```

Comment:
The fedora install script contains a section which masks several services.

Do we perhaps need to mask some services in the Ubuntu build as well? From cursory view of the CI log I see errors:
https://github.com/ovn-org/ovn-fake-multinode/actions/runs/6695210331/job/18190262467?pr=85#step:5:500

Reply:
Thanks for noticing. I though that I'm skipping `systemd` installation, but it seems that it's a dependency for `init` package. I'll add the service masking.